gplog/gplog.go

```diff
@@ -390,7 +391,12 @@ func Fatal(err error, s string, v ...interface{}) {
 	message := ""
 	stackTraceStr := ""
 	if err != nil {
-		message += fmt.Sprintf("%v", err)
+		if _, ok := err.(*pgconn.PgError); !ok {
```

Comment:
I think this change is assuming the error is always a postgres error, but that's definitely not the case for gpbackup. I think we need a case for handling regular errors too